src/components/members/form.tsx

```
@@ -1,203 +1,323 @@
-                evangelism: selectedMember.team.includes('evangelism'),
-                holisticteam: selectedMember.team.includes('holisticteam'),
-                eventOrganization: selectedMember.team.includes('eventOrganization'),
-                telegram: selectedMember.telegram
-            })
-        }
-    }, [selectedMember])
-    return (
-        <div>
-            <form onSubmit={_form.onSubmit(values => handleSubmit(values))}>
+    
+    closeModal && closeModal()
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)